ui/client/components/graph/node-modal/editors/expression/EditableExpression.tsx

```diff
@@ -3,84 +3,84 @@ import ProcessUtils from "../../../../../common/ProcessUtils"
 import {dualEditorMode, editors, editorTypes} from "./EditorType"
 import SwitchIcon from "./SwitchIcon"
 import FixedValuesEditor from "./FixedValuesEditor"
-import _ from "lodash";
-import {$TodoType} from "../../../../../actions/migrationTypes";
+import _ from "lodash"
 
 type Props = {
-  fieldType: string
-  expressionObj: $TodoType
-  showSwitch: boolean
-  renderFieldLabel: Function
-  fieldLabel: string
-  readOnly: boolean
-  rowClassName?: string
-  valueClassName?: string
-  param?: $TodoType
-  values?: Array<$TodoType>
-  editorName?: string
-  fieldName?: string
-  isMarked?: boolean
-  validators?: Array<$TodoType>
-  showValidation?: boolean
-  onValueChange: Function
+  fieldType: string,
```

Comment:
Why commas instead of nothing?

Reply:
types and interfaces, looks similar to objects ({}), are often created from objects, can be formatted in one line. commas are required for objects and separator is required for one liners.